x-pack/plugins/console_extensions/spec/overrides/xpack.rollup.put_job.json

```diff
@@ -0,0 +1,19 @@
+{
+  "xpack.rollup.put_job": {
+    "methods": [
+      "PUT"
+    ],
+    "patterns": [
+      "_xpack/rollup/job/{id}"
```

Comment:
Any reason "methods" and "patterns" are defined both here and in the generated file?